.ci/jenkins/dsl/jobs.groovy

```diff
@@ -178,13 +178,19 @@ Map getMultijobPRConfig(JenkinsFolder jobFolder) {
                 ]
             ], [
                 id: 'optaweb-vehicle-routing',
-                repository: 'optaweb-vehicle-routing'
+                repository: 'optaweb-vehicle-routing',
+                env: [
+                    MIGRATE_TO_9: Utils.isMainBranch(this) && isStream8()
```

Comment:
The migration needs to happen only for the PR targeting the `main` branch.

I assume that `Utils.isMainBranch()` checks https://github.com/kiegroup/optaplanner/blob/main/.ci/jenkins/config/main.yaml#L10 and not if the branch name is `main`.

Thus, it would define `MIGRATE_TO_9=true` even for the `9.x` jobs if I didn't add the `isStream8()`.

@radtriste am I correct?